server/server.go

```diff
@@ -298,6 +299,16 @@ func (s *Server) Serve(ctx context.Context) error {
 		return err
 	}
 	service := openService(ctx, s.BoltPath, layoutBuilder, sourcesBuilder, kapacitorBuilder, logger, s.useAuth())
+
+	// Add any new sources and kapacitors as specified via server flag
+	if err := chronograf.NewSources(ctx, service.SourcesStore, service.ServersStore, s.NewSources, logger); err != nil {
```

Comment:
Interesting idea. Seems like that might create a race condition where a source and Kapacitor server could be added after the server serves up the client, and thus the user may not see these newly added sources?